src/components/404.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+
+function PageNotFound(props) {
+  return (
+    <section className="page_404">
+      <div className="container">
+        <div className="four_zero_four_bg">
+          <h1 className="text-center ">404</h1>
+        </div>
+
+        <div className="contant_box_404">
+          <h3 className="h2">Looks like you are lost</h3>
+
+          <p>We dont have the page you are looking for</p>
+          <Button className="btn-one" onClick={() => { props.history.push('/'); }}> Go home</Button>
```

Comment:
@BrianSerem, kindly destructure props here and make sure it is tested as it has an onClick action.